contracts/apps/disputable/IDisputable.sol

```diff
@@ -0,0 +1,37 @@
+/*
+ * SPDX-License-Identitifer:    MIT
+ */
+
+pragma solidity ^0.4.24;
+
+import "./IAgreement.sol";
+import "../../common/IForwarder.sol";
+import "../../lib/token/ERC20.sol";
+import "../../lib/standards/ERC165.sol";
+
+
+contract IDisputable is IForwarder, ERC165 {
+    bytes4 internal constant ERC165_INTERFACE_ID = bytes4(0x01ffc9a7);
+    bytes4 internal constant DISPUTABLE_INTERFACE_ID = bytes4(0xef113021);
```

Comment:
Just checking, do we have a ERC165 bytes generator somewhere?

I feel like this is normally a trivial thing based on the ABI but so many EIPs have discussions around their ERC165 interfaces so I wasn't sure.

Reply:
Added one and a few tests for it :)